independent-publisher-connectors/Microsoft MVP/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3143 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft MVP",
+    "description": "The MVP Award Program is a community - and you\u2019ve always been there for each-other! A lot of you want to help us make this community better, including creating a better experience for submitting contributions. The MVP Connector will directly enable you to use your own ideas to create and enhance your own MVP experience.",
```

Comment:
@ahmadn82 Please remove unicodes from the all the connector files as they may not be supported in all the Power Automate pages.